### ISSUE TITLE
Support for Injection of multiple views at the same time?

### ISSUE THREAD

**Issue by approce** [johnno1962/InjectionIII#270]
Right not experiencing an issue that only one injected view has last updates from the codebase. Is there any way to "push" multiple updated SwiftUI views so all of them get new implementation?

PS: Thank you for your project, it saves me a huge amount of time every day! Appreciate it

**Comment by johnno1962** [comment]
Thanks! Nice to hear. Have you added an observed object in each view you want to update. It should all just eb automatic:
```Swift
        .eraseToAnyView()
    }

    #if DEBUG
    @ObservedObject var iO = injectionObserver
    #endif
```

**Comment by approce** [comment]
Yup, I've added an observed object in each view and performed 'eraseToAnyView' but nothing has changed. I've also just updated to the last version of injectionIII.

![image](https://user-images.githubusercontent.com/11144422/95442936-a2582f80-0964-11eb-90ad-b9db22d6dcb4.png)
![image](https://user-images.githubusercontent.com/11144422/95442969-aa17d400-0964-11eb-81ad-17564d7fd642.png)

Here is a demo project with these two simple views: https://github.com/approce/injectionIII-demo

The only way for me to get a hot reload is to create Preview with 'injected' method which will replace the root view controller but it feels that I am not using right tool:
![image](https://user-images.githubusercontent.com/11144422/95443139-e51a0780-0964-11eb-8c91-4d99f2ceb9f7.png)

What am I doing wrong?
